### DISCUSSION TITLE
About GPU remote arvester latency problem

### DISCUSSION THREAD

**Discussion by murcielargo** [Chia-Network/chia-blockchain#16499] [Support]
I have 6 servers, each of which is equipped with 32 18TB hard drives. I have prepared 2 NVIDIA GeForce RTX 3070 GPUs for decompression purposes. My local GPU harvester works fine, but the remote harvester has a very high latency of 5s to 60s. I would like to seek advice on how to resolve this issue. Thank you very much！

**Comment by BrandtH22** [comment]
Hey @murcielargo , are you seeing any errors in the logs for either the harvester or the farmer?

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
